website/blog/2018-01-10-1.10.0.md

```diff
@@ -380,7 +380,7 @@ We've fixed up the behavior of the CLI when it comes to ignored files. Previousl
 [@j-f1]: https://github.com/j-f1
 [@thorn0]: https://github.com/thorn0
 [@vjeux]: https://github.com/vjeux
-[#2097]: https://github.com/prettier/prettier/issue/2097
+[#2097]: https://github.com/prettier/prettier/issues/2097
```

Comment:
Oh that's annoyingly inconsistent... It's `/pulls` for the PR list, `/pull/:number` for a PR, then `/issues` for the issues list, and `/issues/:number` for an issue.